uploadNeuroDB/NeuroDB/MRI.pm

```diff
@@ -401,7 +404,7 @@ Returns: Textual name of scan type
 
 sub identify_scan_db {
 
-    my  ($psc, $subjectref, $tarchiveInfoRef, $fileref, $dbhr,$minc_location
+    my  ($psc, $subjectref, $tarchiveInfoRef, $fileref, $dbhr, $db, $minc_location
```

Comment:
why are you keeping the `$dbhr` here even within the one function you are refactoring?

Reply:
never mind my comment, I guess we need it for some of the queries in the function